GitUI/CommandsDialogs/RepoHosting/ForkAndCloneForm.cs

```diff
@@ -438,11 +438,11 @@ private void UpdateCloneInfo(bool updateCreateDirTB = true, bool updateProtocols
 
             if (repo != null)
             {
-                bool multipleProtocols = repo.SupportedCloneProtocols.Length > 0;
+                bool multipleProtocols = repo.SupportedCloneProtocols.Any();
```

Comment:
Any particular reason?
AFAIK accessing a property is faster than doing a method call, because inside it would do the same property check.

Reply:
I find it more expressive (and since especially here, we don't care about performance)